packages/manager/modules/pci/src/projects/project/serving/namespace/infos/infos.html

```diff
@@ -0,0 +1,78 @@
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+<div class="row d-lg-flex">
+    <div class="col-md-4">
+        <oui-tile data-heading="{{ ::'pci_projects_project_serving_namespace_infos_namespace_information_management' | translate }}">
+            <oui-tile-button
+                data-ui-sref="pci.projects.project.serving.namespace.models.add">
```

Comment:
Could this be injected through one of the routing's resolve as it is reliant on it ?

Reply:
1197e33cc8b1ea93fe87171109e2836b7495605b